src/project:kbc/Login.java

```diff
@@ -0,0 +1,85 @@
+import javafx.application.Application;
+import javafx.scene.control.ComboBox;
+import javafx.collections.ObservableList;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.scene.Scene;
+import javafx.scene.text.Text;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import javafx.scene.control.Button;
+import javafx.scene.layout.FlowPane;
+import javafx.geometry.Pos;
+import javafx.event.ActionEvent;
+
+public class Login extends Application{
```

Comment:
maybe rename this LoginView, as we may have a LogInController class to go with this to interface with the 'data' classes. If we want to use MVC for this.